backend/app/Http/Controllers/User/ShowEditUserController.php

```diff
@@ -27,19 +29,50 @@ class ShowEditUserController extends Controller
      */
     public function __invoke(): View|Factory
     {
-        $userId = Auth::id();
-
-        $user = User::where('id', $userId)->first();
+        /**
+         * @var User
+         */
+        $user = Auth::user();
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Call to static method user() on an unknown class App\Http\Controllers\User\Auth.